tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
+			async ( { page } ) => {
+				await test.step(
+					'Navigate to the Payments Overview screen',
+					async () => {
+						await page.goto(
+							'/wp-admin/admin.php?page=wc-admin&path=/payments/overview'
+						);
+					}
+				);
+
+				await test.step(
+					'Observe the total account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const totalBalanceValue = page.getByLabel(
+							'Total balance',
+							{
+								exact: true,
+							}
+						);
+
+						// Match the total balance value to the USD format $1*
+						// Intentionally not expecting a specific value to allow for different values in different environments.
+						await expect( totalBalanceValue ).toHaveText( /\$\d+/ );
+					}
+				);
+
+				await test.step(
+					'Observe the available account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const availableFundsValue = page.getByLabel(
+							'Available funds',
+							{
+								exact: true,
+							}
+						);
+						await expect( availableFundsValue ).toHaveText(
+							/\$\d+/
+						);
+					}
+				);
+			}
+		);
+	}
+);
+
+test.describe(
```

Comment:
Just to confirm, this `test.describe` block will surely have more than one test, right?

Reply:
At this stage, it has just one test. I could draft out another scenario, but this one might be sufficient since the user flow is fairly straightforward.
* Select a currency
* Check the balance of the selected currency
* Select a different currency
* Check the balance of the newly selected currency

Are you thinking that we should keep this to a single `test()` rather than `test.describe()` if there is only one in the group?

Reply:
Thanks for the context!

> Are you thinking that we should keep this to a single test() rather than test.describe() if there is only one in the group?

Yes, if I'm not missing anything, the `describe` block only servers to group tests. This is useful for different reasons:
- It contributes to tests maintenance.
- Makes tests easier to identify and event to understand if it provides additional context (by annotations, or even with the block name).
- It serves to reuse `beforeAll`, `beforeEach`, `afterEach` and `afterAll` hooks for certain tests that apply for it.

If the decision of splitting the spec file into 2 describe blocks, each one containing just 1 test, doesn't fit any of the criteria listed above (or any other meaningful one I might be missing) then I think we might be over complicating this and maybe leaving a single describe block that serves for both a single deposit account currency test and a multiple one would be enough (just following the [YAGNI](https://www.geeksforgeeks.org/what-is-yagni-principle-you-arent-gonna-need-it/) and [KISS](https://en.wikipedia.org/wiki/KISS_principle) principles here).

Otherwise, this is the initial stage of incremental work so if we are sure we will cover more scenarios for single/multiple deposit currency accounts, it's totally justified to leave it as it is right now, since we can justify it by the reasons mentioned above.

Reply:
> single describe block that serves for both a single deposit account currency test and a multiple one would be enough

Totally agree. I don't see a reason to have 2 separate describe groups currently – better to split it out later if needed.

I've opted for a single `describe` with the two child tests in 79a083d:

`test.describe` Merchant account balance overview
→ `test` View the total and available account balance for a single deposit currency `@critical`
→ `test` View the total and available account balance for multiple deposit currencies `@critical` `@todo`